node/src/chain_spec/turing.rs

```diff
@@ -0,0 +1,188 @@
+use hex_literal::hex;
+
+use cumulus_primitives_core::ParaId;
+use sc_service::ChainType;
+use sp_core::{crypto::UncheckedInto, sr25519};
+
+use crate::chain_spec::{
+	get_account_id_from_seed, get_collator_keys_from_seed, validate_allocation, Extensions,
+};
+use primitives::{AccountId, AuraId, Balance};
+use turing_runtime::{
+	CouncilConfig, SudoConfig, ValveConfig, DOLLAR, EXISTENTIAL_DEPOSIT, TOKEN_DECIMALS,
+};
+
+static TOKEN_SYMBOL: &str = "TUR";
+const SS_58_FORMAT: u32 = 51;
+const TOTAL_TOKENS: u128 = DOLLAR * 1_000_000_000;
+static RELAY_CHAIN: &str = "rococo-local";
+static TURING_RELAY_CHAIN: &str = "rococo-testnet";
```

Comment:
wait, it's rococo? shouldn't it be kusama?

Reply:
this spec file is just the shell. Will need to fix it when we know what we want

Reply:
https://github.com/paritytech/polkadot/blob/release-v0.9.16/node/service/res/kusama.json